rryb77/glassdale-pd
# Added the ability to save notes to the JSON server

## Changes:

Added the ability to save notes to the JSON server.

## To Test:

Populate the notes form and hit the save button. Make sure the form is cleared, and the data is saved in the server.
